docs/build-win.md

```diff
@@ -0,0 +1,292 @@
+
+```shell
+cd VCPKG_ROOT
+vcpkg install openssl --triplet x64-windows
+```
+
+The next step is to integrate `vcpkg` with the build system, so that `CMake` can locate `openssl` library.
+
+```shell
+vcpkg integrate install
+```
+
+CMake will be able to find openssl given the following option is provided:
+
+```shell
+-DCMAKE_TOOLCHAIN_FILE=%VCPKG_ROOT%\\scripts\\buildsystems\\vcpkg.cmake
+```
+
+##### 2.1.1.2. Using Installer (Windows)
+
+The 64-bit OpenSSL package for windows can be downloaded using
+the following link: [Win64OpenSSL_Light-1_1_1c](http://slproweb.com/download/Win64OpenSSL_Light-1_1_1c.exe).
+
+**Note!** The last letter or version number may be changed and older versions may become no longer available. In that case found the appropriate installer here: [Win32OpenSSL](http://slproweb.com/products/Win32OpenSSL.html).
+
+Download and run the installer. The library is expected to be installed in `C:\Program Files\OpenSSL-Win64`. Add this path to the user's or system's environment variable `PATH`.
+
+It's expected to be installed in `C:\OpenSSL-Win64`. Note that this version is most likely compiled for Visual Studio 2013. For other versions please follow instructions in Section **2.2.2 Build OpenSSL from Sources**.
+
+##### 2.1.1.2. Build from Sources
+
+Download and compile the sources from the [website](https://github.com/openssl/openssl). The instructions for compiling on Windows can be found here: [link](https://wiki.openssl.org/index.php/Compilation_and_Installation#Windows).
+
+**Note!** `ActivePerl` and `nasm` are required to build OpenSSL.
+
+#### 2.1.2. Install MbedTLS
+
+`MbedTLS` source code can be downloaded from the [website](https://tls.mbed.org/download).
+
+`MbedTLS` comes with `cmake` build system support. Use the `CMAKE_INSTALL_PREFIX` variable to specify the directory that will contain the `MbedTLS` headers and libraries. Note that building `MbedTLS` as a DLL is broken in version 2.16.3. You have to link it statically.
+
+#### 2.1.3. Install LibreSSL
+
+`LibreSSL` has header files that are compatible with OpenSSL, `cmake` can use it like OpenSSL with little configuration. The source code and binaries can be downloaded from here: [link](https://ftp.openbsd.org/pub/OpenBSD/LibreSSL/).
+
+Since there have been no new Windows builds since 2.6.0, you must build a new version yourself. LibreSSL comes with `cmake` build system support. Use the `CMAKE_INSTALL_PREFIX` variable to specify the directory that will contain the LibreSSL headers and libraries.
+
+### 2.2. Threading Library
+
+SRT can use one of these two threading libraries:
+
+- C++11 threads (SRT v1.5.0 and above) - recommended for Windows
+- `pthreads` (default)
+
+**Note!** `pthreads` are available by default on POSIX platforms like Linux and MacOS.
+This step is only required on Windows Platforms.
+
+#### 2.2.1. Using C++11 Threading
+
+Specify the CMake option `-DENABLE_STDCXX_SYNC=ON` ti enable C++11 Threading,
+and exclude the `pthreads` dependency.
```

Comment:
"to" (typo).

I'd rather say:

If you want to use standard C++ threading library (available since C++11), you need to use the `-DENABLE_STDCXX_SYNC=ON` CMake build option. This way there will be also no external dependency on the threading library.